sdk/common/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java

```diff
@@ -1,59 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.sdk.internal;
-
-import io.opentelemetry.sdk.common.Clock;
-import javax.annotation.concurrent.Immutable;
-
-/**
- * This class provides a mechanism for calculating the epoch time using {@link System#nanoTime()}
- * and a reference epoch timestamp.
- *
- * <p>This is needed because Java has millisecond granularity for epoch times and tracing events are
- * recorded more often.
- *
- * <p>This clock needs to be re-created periodically in order to re-sync with the kernel clock, and
- * it is not recommended to use only one instance for a very long period of time.
```

Comment:
do you think we should bring this comment over from here to the new class? I'm not sure I understand exactly what the problem is that's being called out here. I believe that `nanoTime()` is guaranteed to be monotonic, so I'm not sure what this is getting at, TBH.

Reply:
Or was this comment about the fact that `now()` and `nanoTime()` weren't really coordinated and consistent, maybe?

Reply:
It's about the fact that e.g., if you have a global MonotonicClock and just used that forever, if the user changed the time on their computer (or NTP did or whatever), the MonotonicClock wouldn't respect it since it doesn't access wall time after initialization. But since this is basically an implementation detail and we don't actually do anything special, we create a MonotonicClock on the start of the trace, figured it's not such a useful comment.